packages/core/src/Source.ts

```diff
@@ -36,6 +37,7 @@ export default class Source {
   #ignorePages: string[];
   #workflows: SourceWorkflow[] = [];
   #schedule: SourceSchedule;
+  #pluginErrors: any = {};
```

Comment:
Will update the type of this when we have identified what we want to do with the errors.  They are logged to the console but it would be nice to provide an API that can be used to see the errors

Reply:
might be worth leaving this till after we merged in Next 13 changes... as right now, it's easy to rebase that branch, when we confine other changes to core and not change any of the UI

Reply:
That's fine...just pushed a change that will allow you to see the plugin errors when using the list sources admin API.  Should be good enough for us to diagnose problems.